src/main/java/com/ddd/moodof/application/dto/BoardDTO.java

```diff
@@ -15,6 +17,8 @@
     public static class CreateBoard {
         private Long previousBoardId;
         private Long categoryId;
+        @Max(value = 20, message = "보드의 최대 글자수는 20자 입니다.")
```

Comment:
```suggestion
        @Max(value = 20, message = "보드의 최대 글자수는 20자 입니다.")
        @Min(value = 1, message="보드의 최소 글자수는 1자 입니다.")
```

최소글자에 대한 처리도 있어야하지 않을까요. 공백처리를 지금 프론트엔드딴에서 처리를 하고 있는건지 모르겠네용!

Reply:
Max 밑에 `@NotBlank`가 있기 때문에 Min을 추가할 필요는 없다고 생각했습니다.

Reply:
@NotBlank를 확인을 제대로 못했네요. @NotBlank가 있다면 굳이 추가할 필요는 없어보이네요.